src/renderer_mtl.mm

```diff
@@ -4662,6 +4662,27 @@ static void setTimestamp(void* _data)
 								{
 								case Binding::Image:
 								{
+									if (bind.m_access == Access::ReadWrite && 0 == (g_caps.supported & BGFX_CAPS_IMAGE_RW))
+									{
+										BGFX_FATAL(false, Fatal::DebugCheck,
+										"Failed to set image with access: Access::ReadWrite, device is not support image read&write");
+									}
+
+									if (
+                                        (bind.m_access == Access::Read && (0 == (g_caps.formats[bind.m_format] & BGFX_CAPS_FORMAT_TEXTURE_IMAGE_READ)))
+										|| (bind.m_access == Access::Write && (0 == (g_caps.formats[bind.m_format] & BGFX_CAPS_FORMAT_TEXTURE_IMAGE_WRITE)))
+                                        || (bind.m_access == Access::ReadWrite && (0 == (g_caps.formats[bind.m_format] & (BGFX_CAPS_FORMAT_TEXTURE_IMAGE_READ|BGFX_CAPS_FORMAT_TEXTURE_IMAGE_WRITE))))
+                                        )
+									{
+										const char* accessNames[] = {
+											"Access::Read",
+											"Access::Write",
+											"Access::ReadWrite",
+										};
+										BX_STATIC_ASSERT(sizeof(accessNames)/sizeof(accessNames[0]) == Access::Count, "Invalid accessNames");
```

Comment:
Use `BX_COUNTOF(...)`, and move `const char*` static array outside of function scope with the rest of static arrays.

Reply:
OK, I have changed that.